apps/src/code-studio/pd/foorm/editor/form/FoormFormSaveBar.jsx

```diff
@@ -260,24 +261,27 @@ class FoormFormSaveBar extends Component {
         <div style={styles.saveButtonBackground} className="saveBar">
           {this.props.lastSaved &&
             !this.props.saveError &&
-            !this.props.hasJSONError && (
+            !this.props.hasJSONError &&
+            !this.props.hasLintError && (
```

Comment:
It looks like `hasLintError` has to be true everywhere that `hasJSONError` has to be true (and the reverse) -- could we add a little helper function that combines them and use that throughout (eg, hasCodeMirrorError()?)

Reply:
There's a couple places I wanted to specifically not combine them (i.e. the preview can still render if there is a lint error but no json error) but yeah, i could add a helper for the rest of the cases

Reply:
In this component (and the library save bar one) it looks like they're always used in combination, right? I feel like worth having a helper in those?